LICENSE.bin

```diff
@@ -0,0 +1,4712 @@
+  the public domain, and simply represents an additional license granted by
+  the Author.
+
+  -----------------------------------------------------------------------------
+  ** Beginning of "BSD 2-Clause License" text. **
+
+   Copyright (c) 2012, 2013, 2014, 2015, 2016 Gil Tene
+   Copyright (c) 2014 Michael Barker
+   Copyright (c) 2014 Matt Warren
+   All rights reserved.
+
+   Redistribution and use in source and binary forms, with or without
+   modification, are permitted provided that the following conditions are met:
+
+   1. Redistributions of source code must retain the above copyright notice,
+      this list of conditions and the following disclaimer.
+
+   2. Redistributions in binary form must reproduce the above copyright notice,
+      this list of conditions and the following disclaimer in the documentation
+      and/or other materials provided with the distribution.
+
+   THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+   AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+   IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+   ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+   LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+   CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+   SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+   INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+   CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+   ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
+   THE POSSIBILITY OF SUCH DAMAGE.
+
+---
+org.slf4j:jul-to-slf4j
+org.slf4j:slf4j-api
+
+  Copyright (c) 2004-2022 QOS.ch Sarl (Switzerland)
+  All rights reserved.
+
+  Permission is hereby granted, free  of charge, to any person obtaining
+  a  copy  of this  software  and  associated  documentation files  (the
+  "Software"), to  deal in  the Software without  restriction, including
+  without limitation  the rights to  use, copy, modify,  merge, publish,
+  distribute,  sublicense, and/or sell  copies of  the Software,  and to
+  permit persons to whom the Software  is furnished to do so, subject to
+  the following conditions:
+
+  The  above  copyright  notice  and  this permission  notice  shall  be
+  included in all copies or substantial portions of the Software.
+
+  THE  SOFTWARE IS  PROVIDED  "AS  IS", WITHOUT  WARRANTY  OF ANY  KIND,
+  EXPRESS OR  IMPLIED, INCLUDING  BUT NOT LIMITED  TO THE  WARRANTIES OF
+  MERCHANTABILITY,    FITNESS    FOR    A   PARTICULAR    PURPOSE    AND
+  NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+  LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+  OF CONTRACT, TORT OR OTHERWISE,  ARISING FROM, OUT OF OR IN CONNECTION
+  WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
```

Comment:
Do we distributed the third party jar file(e.g., `org.slf4j:slf4j-api`)?  If yes, then we don't have to copy the license in Polaris. The jar has the license. 
<img width="465" alt="Screenshot 2024-12-03 at 9 53 54 AM" src="https://github.com/user-attachments/assets/f4f42fc1-042e-4a58-99a3-799f1e4fdb1b">

Reply:
if we have to include the third party license, can we use the link to save the space like what Iceberg does?
```
This product includes code from Apache Hive.

* Hive metastore derby schema in hive-schema-3.1.0.derby.sql

Copyright: 2011-2018 The Apache Software Foundation
Home page: https://hive.apache.org/
License: https://www.apache.org/licenses/LICENSE-2.0
```
https://github.com/apache/iceberg/blob/feed4e2544b5839fbc2fe040965af3906d053302/LICENSE#L273-L273

Reply:
We ship slf4j in the binary distribution. No need to include the License only copy the notice content (according to alv2 4d section). 

Iceberg is different as it's only source distribution (not binary). So it mentions only the source code coming from other projects. 

Just to be clear: I'm pretty sure of my changes here (especially for the source distribution). To license and notice of the binary distribution are generated by the gradle plugin.

Reply:
> Do we distributed the third party jar file(e.g., `org.slf4j:slf4j-api`)?  If yes, then we don't have to copy the license in Polaris. The jar has the license. 
> 
> <img width="465" alt="Screenshot 2024-12-03 at 9 53 54 AM" src="https://github.com/user-attachments/assets/f4f42fc1-042e-4a58-99a3-799f1e4fdb1b">
> 
> 

That doesn't matter: the notice would have been important but slf4j doesn't have notice

Reply:
> Iceberg is different as it's only source distribution (not binary). So it mentions only the source code coming from other projects.

Should we do that for the Polaris source distribution? Oh, I guess this statement does the job. We should be fine?
```
   This product bundles various third-party components also under the
   Apache Software License 2.0.
```